wgpu/src/api/texture.rs

```diff
@@ -37,7 +37,11 @@ impl Texture {
         }
     }
 
-    /// Creates a view of this texture.
+    /// Creates a view of this texture, specifying a rectangular region of it, and an
+    /// interpretation of its texels.
```

Comment:
You can't actually restrict a view to a rectangular region of a texture. You can only choose which set of subresources you can access.